_posts/2022-02-15-a-quick-opensearch-primer.md

```diff
@@ -0,0 +1,78 @@
+
+![stacked chart example](/assets/media/blog-images/2022-02-15-a-quick-opensearch-primer/stacked-chart.png){: .img-fluid}
+
+This example is a stacked graph of response codes over time. Certain events are configured to trigger an anomaly warning. It’s a short example of what visualizations are capable of.
+
+**Building your own custom search engine** is another use for which OpenSearch is fully capable.  Using the OpenSearch client library gives you access to ranked search, field weighting, and many other features common for search engines.  Results are scored based on the frequency of the terms searched, how many documents contain that term, and how many times they contain it, amongst other criteria. OpenSearch can **weight** the results as well. For example, matches against a document’s title can be considered a more meaningful match than matching terms in a document’s content. 
+
+
+### See for yourself
+
+The quickest way to get hands-on is to bring up an instance of OpenSearch.  Inspect the sample datasets and visualizations to get a feel for it. It is the straightest path to exemplifying what all can be done.  Visit the [downloads](https://opensearch.org/downloads.html) page and follow the instructions to get started. 
+
+After logging in with `admin:admin` you’ll see the home screen.  ![Home Screen](/assets/media/blog-images/2022-02-15-a-quick-opensearch-primer/home.png){: .img-fluid}
+
+
+
+Click ‘Add Data’ in the upper right or ‘Add Sample Data’ in the lower left - you’ll be shown three sample data sets. There’s no harm in installing all three of them.
+
+![Add Sample Data](/assets/media/blog-images/2022-02-15-a-quick-opensearch-primer/add-sample-data.png){: .img-fluid}
+
+
+
+
+There’s nothing like seeing it in action! The sample data sets offered come with dashboards that are excellent examples of the visualizations and how the information is structured. What you see here is the dashboard that comes with some test data for a fake airline company.  Heat maps, stacked graphs, pie charts, gauges and bar charts are available, among others. 
+
+![dashboard example](/assets/media/blog-images/2022-02-15-a-quick-opensearch-primer/dashboard-sample.png){: .img-fluid}
+
+
+
+
+A wide selection of visualizations are available.  Arrange them on a Dashboard to get a view that’s just right.
+
+
+![Visualization Selections](/assets/media/blog-images/2022-02-15-a-quick-opensearch-primer/viz-types.png){: .img-fluid}
+
+
+
+
+
+
+
+### All set, but it’s just the start. 
+
+Hopefully some of the uncertainty about what OpenSearch is and what it does has been cleared up. Its base functionality  has a wide variety of use cases, such as building a custom search engine.  Also, ingesting logs for the ease of searching and aggregating. Both make use of the strong indexing and search library under the hood. OpenSearch Dashboards extends this even further by allowing you to build the exact view you want.  Even ones that can warn you of anomalies in your data. 
```

Comment:
This just repeats what's above, I would talk about other more complex scnearios/example as next steps.